### ISSUE TITLE
0.3pre: Says "KDE Plasma" in System Information

### ISSUE THREAD

**Issue by probonopd** [mszoek/airyx#100]
It says "KDE Plasma" in System Information when in fact (I assume) it is not using KDE Plasma but some custom airyx desktop environment?

**Comment by mszoek** [comment]
Nope, it actually is (parts of) KDE plasma with some additional widgets and clever themes! Some of those parts are likely getting replaced or removed and I'll definitely be replacing the system info - but for now, it is what it is. There's only so many hours available to work on this 🤷🏻‍♀️
